app/javascript/components/activity/producttotals.vue

```diff
@@ -0,0 +1,90 @@
+<template lang="html">
+  <div class="product-totals">
+    <user-input class="mb-3" v-model="user" :include-payment="true"></user-input>
+
+    <spinner class="pb-3 m-auto" size="large" v-if="isLoading" />
+    <div class="table-responsive" v-else>
+      <table class="table table-sm" v-if="orderTotals.length > 0">
+        <thead>
+          <tr>
+            <th scope="col">Product</th>
+            <th scope="col">Aantal keer besteld</th>
+            <th class="text-right" scope="col">Totaalbedrag</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="row in orderTotals">
+            <td>{{row.name}}</td>
+            <td>{{row.amount}} x</td>
+            <td class="text-right">{{doubleToCurrency(row.price)}}</td>
+          </tr>
+        </tbody>
+      </table>
+      <div v-else>
+        Er zijn geen producten verkocht
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import UserInput from '../userinput.vue';
+  import Spinner from 'vue-simple-spinner';
+
+  export default {
+    props: {
+      activity: {
+        required: true,
+      }
+    },
+
+    data() {
+      return {
+        user: {
+          id: 0
+        },
+        orderTotals: [],
+        isLoading: true
+      }
+    },
+
+    created() {
+      this.loadProductTotals();
+    },
+
+    methods: {
+      loadProductTotals() {
+        this.isLoading = true;
+
+        let url = '/activities/'+this.activity+'/product_totals';
+        if (this.user.id > 0) {
+          url += '?user='+this.user.id;
+        }
+        if (this.user.paid_with_cash) {
+          url += '?paid_with_cash=true';
+        }
+        if (this.user.paid_with_pin) {
+          url += '?paid_with_pin=true';
+        }
+        this.$http.get(url).then((response) => {
```

Comment:
can't you pass an params argument to the get method? Then you could do `.get(url, params=this.user)`